Makefile

```
@@ -0,0 +1,135 @@
+	fi
+	@if ! docker info > /dev/null 2>&1; then \
+	  echo "ERROR: docker daemon is not running."; \
+	  echo "  macOS:  open -a Docker"; \
+	  echo "  Linux:  sudo systemctl start docker"; \
+	  exit 1; \
+	fi
+
+# Check that ~/.omnibase/.env (or whatever OMNIBASE_ENV_FILE points to) exists.
+# Fail with a clear remediation path rather than a stack trace from a
+# downstream script.
+_check-env-file:
```

checkmake (0.3.2)

[warning] Target body for "_check-env-file" exceeds allowed length of 5 lines (12).

(maxbodylength)